### DISCUSSION TITLE
Am I the only one who is experiencing popping noise during audio playback with the official Android client?

### DISCUSSION THREAD

**Discussion by HubKing** [jellyfin/jellyfin#8537] [Q&A]
I have tested it with several songs, and all of them have popping noise when played with the official Android client. I have used another non-official Android client, and it had no popping noise. This is not because of battery optimisation, because I have disabled it as the app requested, and the noise is there even when the official client is the foreground app with the screen on. The non-official client did not request disabling battery optimisation in the first place and it had no popping noise even when it was playing the music in the background.

I have searched the Issues, but I could find only one entry https://github.com/jellyfin/jellyfin/issues/3055 , but that said "significant distortion". This may be subjective, but probably he meant something different, because I would not call such popping noises "significant distortion", so I am not sure if I am the only one who is experiencing popping noise in the official Android client.

If you are playing music with the official Android client, is the sound fine without a weird noise?

If it is of any worth, my Android version is 12.

PS: I have tested it on another Android device (Android 11, midrange CPU lower than the aforementioned device). This time, the official client had no popping noise when I did not touch the device, but when the player is playing music in the background and I every time launched the "Google" app, the music stuttered. I tested the same thing with the aforementioned third-party client, and it did not stutter.

**Comment by DJPlaya** [comment]
I only play video files on my device, so idk it works, but on android, you can set the media player to use in your profile settings.
I use and recommend VLC player, the integrated exoplayer never worked that good for me either(not even external).

**Reply by HubKing** [reply]
Well, by "audio", I meant audio-only files like MP3, or FLAC, not the audio part of video files. I have already set it to play video using VLC, but playing MP3 with an external app is not feasible, unlike video files, because music files are short. I guess, if this is a known issue, I have to use that third-party music-only Jellyfin client ("Finamp" or something).

**Comment by midoreigh** [comment]
2026. Samsung S24 Ultra. Same issue. When I play the same file in CX File explorer, no issue.

**Reply by midoreigh** [reply]
No noises in this player

https://play.google.com/store/apps/details?id=com.unicornsonlsd.finamp
